index.js

```diff
@@ -199,11 +202,23 @@ class Channel {
       encoding,
       onmessage,
       recv (state, session) {
+        s.tracer.trace('recv', {
+          id: s.id,
+          protocol: s.protocol,
+          type
+        })
+
         return session._track(m.onmessage(encoding.decode(state), session))
       },
       send (m, session = s) {
         if (session.closed === true) return false
 
+        s.tracer.trace('send', {
+          id: s.id,
+          protocol: s.protocol,
```

Comment:
super nit but the order should be protocol, id, type for both send and recv

Reply:
Makes sense, fixed